Framework/CurveFitting/test/Algorithms/ConvertToYSpaceTest.h

```diff
@@ -63,9 +63,9 @@ class ConvertToYSpaceTest : public CxxTest::TestSuite {
     TS_ASSERT_DELTA(-1.670937938, ySpOutX[npts / 2], 1e-08);
     TS_ASSERT_DELTA(17.99449408, ySpOutX.back(), 1e-08);
     // Y
-    TS_ASSERT_DELTA(-0.01152733, ySpOutY.front(), 1e-08);
-    TS_ASSERT_DELTA(5.56667697, ySpOutY[npts / 2], 1e-08);
-    TS_ASSERT_DELTA(-0.35141703, ySpOutY.back(), 1e-08);
+    TS_ASSERT_DELTA(0.48319776, ySpOutY.front(), 1e-08);
```

Comment:
The algorithm itself doesn't. However, the test makes use of `ComptonProfileTestHelpers::createTestWorkspace` which does make use of MersenneTwister, and thus it changes the contents of the workspace it uses for the algorithm. This in turn requires the test values to be changed.